File: SDKs/logoraisr.com/v1/python/src/sdk/models/shared/color.py
from dataclasses import dataclass, field
from typing import Optional
from dataclasses_json import dataclass_json


@dataclass_json
@dataclass
class Color:
    blue: Optional[int] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'blue' }})
    green: Optional[int] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'green' }})
    hex: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'hex' }})
    number_of_pixel: Optional[int] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'number_of_pixel' }})
    percentage: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'percentage' }})
    red: Optional[int] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'red' }})
    

File: SDKs/logoraisr.com/v1/typescript/src/sdk/models/shared/color.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class Color extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=blue" })
  blue?: number;

  @SpeakeasyMetadata({ data: "json, name=green" })
  green?: number;

  @SpeakeasyMetadata({ data: "json, name=hex" })
  hex?: string;

  @SpeakeasyMetadata({ data: "json, name=number_of_pixel" })
  numberOfPixel?: number;

  @SpeakeasyMetadata({ data: "json, name=percentage" })
  percentage?: string;

  @SpeakeasyMetadata({ data: "json, name=red" })
  red?: number;
}
